contracts/Otoken.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity =0.6.0;
+
+contract Otoken {
```

Comment:
Otoken should be ERC20Detailed, that way the name, symbol etc are all taken care of in that

Reply:
I use `ERC20UpgradeSafe` as suggested in this post: [Writing Upgradeable Contracts](https://docs.openzeppelin.com/upgrades/2.8/writing-upgradeable) because we need the ERC20 to have an init function instead of a constructor.